### DISCUSSION TITLE
The docker dont give me root acces

### DISCUSSION THREAD

**Discussion by Hackermanswitch** [coder/code-server#3852] [Q&A]
Hello Community,

Im running code-server on my unraid system and installed it from this repo: docker repo: linuxserver's Repository.

Now everything works great but I have a few issues with the debugging, which I believe are caused by the same issue.
the issues I got are:

- in the folder of my script are the following directories that aren't seen by the script, modules & binaries.
- I dont have root acces in the terminal it logs as followed
    `abc@0f20009e3b3f:/mnt/disks/NVMe/opt/PythonScripts`

I think this has to do with the web ui doesn't have root acces and there fore can't use the directories.
<img width="500" alt="Schermafbeelding 2021-07-26 om 10 39 48" src="https://user-images.githubusercontent.com/67789649/126960125-5e7e741e-5bdf-47a8-a164-da63e4941457.png">

does anyone know what im doing wrong? 
if u need more info please let me know its essential for me to debug.

**Comment by jsjoeio** [comment]
>  I have a few issues with the debugging

Hmm...can you tell us more about this? It sounds like code-server works normally but debugging is not working, is that right?

What exactly are you doing when you say "debugging"? (Intentionally asking vague questions to get a better understanding of your environment and how you're using code-server)

**Reply by Hackermanswitch** [reply]
Hello, 

First of all thanks for your response. Ill try to explain it as clear as possible.

Step 1:
I downloaded the docker from the repo as shown below:
<img width="1247" alt="Schermafbeelding 2021-07-27 om 09 38 21" src="https://user-images.githubusercontent.com/67789649/127115216-86022b5b-dbb8-4414-b737-3f105e48094c.png">

Step 2: 
I opened a folder in Code studio called:
/mnt/disks/NVMe/opt/PythonScripts/name of the folder.

Step 3:
I installed a python needed tools in Code studio as shown below:
<img width="918" alt="Schermafbeelding 2021-07-27 om 09 43 43" src="https://user-images.githubusercontent.com/67789649/127115963-480fdda4-64cb-4b58-913c-1f67dd36361c.png">

everything so far is okay and works the way it should.

Step 4:
Now I want to start debugging so I start the debugging tool like this:
<img width="416" alt="Schermafbeelding 2021-07-27 om 09 46 26" src="https://user-images.githubusercontent.com/67789649/127116934-adf4e026-7a1f-4347-94af-d2059e4bac66.png">

Step 5:
I get the following error:
<img width="620" alt="Schermafbeelding 2021-07-27 om 09 51 47" src="https://user-images.githubusercontent.com/67789649/127117240-4527f11a-23a1-4b3c-8be9-96c32a593982.png">
 
which is weird cause I loaded the whole folder, the folder structure looks as following:
<img width="1024" alt="Schermafbeelding 2021-07-27 om 09 55 07" src="https://user-images.githubusercontent.com/67789649/127117982-7b4771b2-62d9-40c9-b0f5-d3b9ba59c7e3.png">

like I said I guess it has something to do with the root privilege because it logs me In as a random user:
<img width="362" alt="Schermafbeelding 2021-07-27 om 09 58 50" src="https://user-images.githubusercontent.com/67789649/127118148-85d58278-d501-4e02-9dc6-efbbc9a07144.png">




I'll hope this clear things up a bit.

**Reply by jsjoeio** [reply]
Thanks for all the details! I wish I had a better strategy for helping you figure this out. 

Is it possible for you to install the extensions one at a time to figure out which one is results in this error.

I'm not familiar with whatever UI/interface you're using to create this but do you need to toggle on privileged? 

![image](https://user-images.githubusercontent.com/3806031/127195122-dc6f8b24-2061-417f-9506-e9f41fd51057.png)
